database.py

```
@@ -0,0 +1,222 @@
+                        "memory_limit": row[6],
+                        "status": row[7],
+                        "expires_at": row[8],
+                        "url": row[9],
+                        "created_at": row[10]
+                    }
+                    for row in rows
+                ]
+    
+    async def update_userbot(self, bot_id: int, **kwargs):
+        """Обновить юзербота"""
+        set_clause = ", ".join([f"{key} = ?" for key in kwargs.keys()])
```

Ruff (0.12.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)